src/strategies/layers/fees/external/ExternalFees.sol

```diff
@@ -51,7 +51,8 @@ abstract contract ExternalFees is BaseFees, ReentrancyGuard, Initializable {
     Fees memory fees = _getFeesOrFailIfSenderCantWithdraw();
     (address[] memory allTokens, uint256[] memory currentBalances) = _fees_underlying_totalBalances();
     _updateFeesForWithdraw({ tokens: tokens, withdrawAmounts: toWithdraw, currentBalances: currentBalances, fees: fees });
-    IEarnStrategy.WithdrawalType[] memory types = _fees_underlying_withdraw(0, tokens, toWithdraw, recipient);
+    _fees_underlying_withdraw(0, tokens, toWithdraw, recipient);
+    IEarnStrategy.WithdrawalType[] memory types = _fees_underlying_supportedWithdrawals();
```

Comment:
Let's move this after the if on the next line, since we won't need it if we revert